tech/languages/python/micropython.md

```diff
@@ -32,23 +32,25 @@ and then start an interactive MicroPython session.
 
 ```
 $ micropython
-MicroPython v1.7 on 2016-04-20; linux version
+MicroPython v1.12 on 2020-04-15; linux version
 Use Ctrl-D to exit, Ctrl-E for paste mode
 >>>
 ```
 
 Using MicroPython on your computer makes sense only for some basic testing
-purposes, but the main goal is to control embedded devices.
+purposes.
+Its main goal is to control embedded devices.
 
 ## Mu - simple IDE for BBC micro:bit
 
 Micro:bit tries to be as simple to use as possible, so its developers decided
-to make their own IDE named *Mu*. *Mu* is designated for kids to
-learn the basics of programming. Your device is mounted automatically in Fedora,
-so you only need one click to flash your micro:bit with your new app or to
-start REPL.
+to make their own IDE named *Mu*. *Mu* is designed for beginners to
+learn the basics of programming.
+When coding in *Mu*, you only need one click to flash (upload) your app
+to your micro:bit or to start the REPL, an interactive Python session.
+Your device is mounted automatically in Fedora.
```

Comment:
There's no need to split the text due to line lenght, because the html pages do this for you.